src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -360,6 +361,8 @@
     FLUSH_LOCAL_SCHEMA_CHANGES("cassandra.test.flush_local_schema_changes", "true"),
 
     TOMBSTONE_HISTOGRAM_TTL_ROUND_SECONDS("cassandra.streaminghistogram.roundseconds", "60"),
+
+    USE_OA_SSTABLE_FORMAT("cassandra.use_oa_sstable_format", BigFormat.TTL_MODE.LEGACY_TTL_2038.toString())
```

Comment:
I understand that this property will go away or change to a boolean as soon as we have the full downgradability thing and/or a way to allow users decide the sstable format. So perhaps we should add a comment about that and mark the property with `@Deprecated`.

Reply:
Whatever the new downgradability thing comes it will use whatever mechanics it decides. I still think it won't be a boolean mainly bc during mixed clusters you need a way to dynamically check the cluster status. That check could be in the hot path, such as it's here, so you need a 3rd value to signal the cluster is fully migrated and we can spare that check.

I can add the comment. But I wouldn't mark it as deprecated until that ticket is complete.

Reply:
Solved differently in later commits